### ISSUE TITLE
CLI command to healthcheck a node

### ISSUE THREAD

**Issue by hairyhum** [build-trust/ockam#3790]
Ockam nodes communicate with each other using secure channels over transports. To be accessible it's not enough for a node to just be running, it should allow secure channels to be established and messages to be routed through.

To check that a node is fully initialized and ready to process messages we can use a simple Ockam program which will connect via secure channel and then call some special worker over the channel. This worker can be a simple echoer replying with the same payload.

This program can be implemented as a CLI command like this:
```
ockam healthcheck --to /route/to/node/api/service/healthcheck_worker
```

This command should establish a secure channel to `/route/to/node/api` and then send some sort of `ping` message to `healthcheck_worker` worker and expect an echo reply.
If reply is received - the command should terminate with exit status `0`.

If there is an error, it should be printed to the STDERR and command should terminate with non-zero exist status.

It would be better if command process is terminated on `EOF` so it can be used by such systems as Elixir ports: https://github.com/build-trust/ockam/issues/3701

**Comment by miles170** [comment]
Can't we use the `echo` service to do the same thing?

```bash
$ ockam message send ping --to /route/to/node/service/echo
ping
```

**Comment by hairyhum** [comment]
> Can't we use the echo service to do the same thing? 

If there was a way to pass this message through a secure channel - yes. But currently you would have to create a node, start a secure channel on it using `ockam secure-channel create --from /node/n1 --to /destination/...` and then call message send `ockam message send hello --from /node/n1 --to -/service/uppercase`

**Comment by miles170** [comment]
> To check that a node is fully initialized and ready to process messages we can use a simple Ockam program which will connect via secure channel and then call some special worker over the channel. This worker can be a simple echoer replying with the same payload.

If we want to connect via a secure channel, we must have a `from` node, right?

**Comment by hairyhum** [comment]
> If we want to connect via a secure channel, we must have a from node, right?

Yes. And this is harder to do from just a single command since it can have more impact and more error modes then just running the whole flow from the CLI node.

**Comment by tilltonystark** [comment]
Hey!
I'm Satyam, a UI/UX designer and I saw this issue as a type of user experience. So, just wondering if I could help here. 

PS- I've not much experience in any programming language.
